delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/syntax/EventLogSyntax.scala

```diff
@@ -41,31 +41,31 @@ final class EventLogOpts[State, E <: Event](private val eventLog: EventLog[Envel
   def projectEvents[R](projects: Projects, projectRef: ProjectRef, offset: Offset)(implicit
       rejectionMapper: Mapper[ProjectNotFound, R]
   ): IO[R, Stream[Task, Envelope[E]]] =
-    EventLogUtils.projectEvents(projects, eventLog, projectRef, offset)
+    projects.fetch(projectRef).bimap(rejectionMapper.to, _ => eventLog.currentEventsByTag("", offset))
 
   /**
     * Fetch project events for the given module
     */
   def projectEvents[R](projects: Projects, projectRef: ProjectRef, module: String, offset: Offset)(implicit
       rejectionMapper: Mapper[ProjectNotFound, R]
   ): IO[R, Stream[Task, Envelope[E]]] =
-    EventLogUtils.projectEvents(projects, eventLog, projectRef, module, offset)
+    projects.fetch(projectRef).bimap(rejectionMapper.to, _ => eventLog.currentEventsByTag(module, offset))
 
   /**
     * Fetch project events
     */
   def currentProjectEvents[R](projects: Projects, projectRef: ProjectRef, offset: Offset)(implicit
       rejectionMapper: Mapper[ProjectNotFound, R]
   ): IO[R, Stream[Task, Envelope[E]]] =
-    EventLogUtils.currentProjectEvents(projects, eventLog, projectRef, offset)
+    projects.fetch(projectRef).bimap(rejectionMapper.to, _ => eventLog.currentEventsByTag("", offset))
```

Comment:
The use of "" for no module is a bit odd.

Reply:
I agree but this class will go away soon, the problem is that a lot of classes point to each other and there is no real satisfactory way to handle this